### ISSUE TITLE
ACRA 4.7 RC1 PHP server POST empty.

### ISSUE THREAD

**Issue by asamaru7** [ACRA/acra#310]
php server not received post data. php server is php-fpm. "Transfer-Encoding: chunked" not work.
when I remove this line. worked.

HttpRequest.java
```java
urlConnection.setChunkedStreamingMode(0);
```

**Comment by william-ferguson-au** [comment]
This appears to be a problem with servers running PHP version prior to 5.3.0

See http://stackoverflow.com/questions/15913844/setting-setchunkedstreamingmode-in-httpurlconnection-fails-to-deliver-data-to-se

I'm not sure I want to change that in the code. as it would mean that the client will cache the entire request until it is 100% sent.

**Comment by william-ferguson-au** [comment]
If you want to change my mind you'll need to provide some good justification.

**Comment by asamaru7** [comment]
@william-ferguson-au 
i am sorry. I am not good at English.

I tested php version is PHP 5.5.2 and httpd-2.4.7. not prior to 5.3.0.

Tested mod_php with apache : OK
Tested mod_proxy_fcgi(php-fpm) with apache : Fail(Empty body).

This problem occurs when i use a proxy mode.

I changed HttpRequest.java. it's work;
```java
// urlConnection.setChunkedStreamingMode(0);
urlConnection.setFixedLengthStreamingMode(content.getBytes("UTF-8").length);
```

ACRA know content length. therefore acra can use `setFixedLengthStreamingMode`.
How about this?
